CMakeLists.txt

```diff
@@ -172,6 +176,15 @@ if(UNIX AND NOT APPLE)
   endif()
   find_package(X11 COMPONENTS X11_xcb)
 endif()
+
+find_package(Qt6 REQUIRED COMPONENTS Protobuf)
```

Comment:
Using multiple find_package lines for the same library is bad practice. Move Protobuf to QT_OPTIONAL_COMPONENTS

Reply:
I'm really out of my depth here and used a solution I found from examples & tutorials.
I need the first find_package for the Qt6 protobuf package and the second find_package for  protobuf_generate.
If I comment any of them out cmake fails.

Reply:
Done